devtools-ci-index.yaml

```diff
@@ -2683,6 +2683,16 @@
         - timed: '{bdd_test_start_time}'
     <<: *booster_bdd_tests_template
 
+- job-template:
+    name: 'devtools-bdd-test-booster-import-{build_tool}-{test_url}-{cluster}'
+    scenario: 'import-repo'
```

Comment:
scenario should be moved to job definition otherwise it will fail:
```
jenkins_jobs.errors.JenkinsJobsException: scenario parameter missing to format booster-bdd.{scenario}.{build_tool}
```